config/filter.d/phpmyadmin-syslog.conf

```diff
@@ -0,0 +1,17 @@
+# Fail2Ban fitler for the phpMyAdmin-syslog
+#
+
+[INCLUDES]
+
+before = common.conf
+
+[Definition]
+
+_daemon = phpMyAdmin
+
+failregex = ^%(__prefix_line)suser denied: .* \(mysql-denied\) from <HOST>\s*$
```

Comment:
I don't like greedy catch-all (especially before host), also if regex is both sides anchored.

I suspect the user is never something else as `\S+`, but for the case of bothering from script-kiddie (ddosing):
```diff
-failregex = ^%(__prefix_line)suser denied: .* \(mysql-denied\) from <HOST>\s*$
+failregex = ^%(__prefix_line)suser denied: (?:\S+|.*?) \(mysql-denied\) from <HOST>\s*$
```